Common/FeatureFlags.swift

```diff
@@ -171,6 +178,9 @@ extension FeatureFlagConfiguration {
         if UserDefaults.appGroup?.allowDebugFeatures ?? false {
             return true
         }
+        if debugFeaturesEnabled {
+            return debugFeaturesEnabled
+        }
         return false
```

Comment:
Might I suggest just adding this here?  I think it would make it less confusing:
```suggestion
        #if ALLOW_DEBUG_FEATURES_ENABLED
        return true
        #else
        return false
        #endif
```